gabrielgomesml/training_project_client
# Feature/sign-up

## What I did

- Create Sign Up flow using redux.

## How to test

- First of all make sure you are running the server in your machine. You can check the steps here: https://github.com/gabrielgomesml/training_project_server/pull/1
- Open the project and use the command `nvm use`
- Run a `yarn` or `yarn install`
- After that run the command `yarn start` to start the project
- Click in "Cadastre-se" and test the flow of creating a new user.

P.S: I didn't connect the success and failure action in the request, i intend to do this later. Also, no photo adding for now.
